api/apiFunctions.ts

```diff
@@ -320,7 +320,6 @@ export async function createEntry({
 }
 
 /**
-
```

Comment:
Remove this. It will mark the file as being modified when it wasn't

Reply:
that's weird. I don't see that space locally

Reply:
Please address this feedback in your code.